### DISCUSSION TITLE
有大佬做过web端替换占位图怎么操作吗  目前pag能展示  但是使用pagFile.replace之后无法替换占位图

### DISCUSSION THREAD

**Discussion by jan0192** [Tencent/libpag#1868] [General] [answered]
有大佬做过web端替换占位图怎么操作吗  目前pag能展示  但是使用pagFile.replace之后无法替换占位图

**Comment by zhalice2011** [comment, marked as answer]
demo 代码: https://github.com/libpag/pag-web

在线示例: https://pagtest.zhalice.com/pages/replace-image.html
